receiver/purefbreceiver/README.md

```diff
@@ -0,0 +1,51 @@
+# Pure Storage FlashBlade Receiver
+
+| Status                   |                     |
+| ------------------------ |---------------------|
+| Stability                | [in-development]    |
+| Supported pipeline types | metrics             |
+| Distributions            | [contrib]           |
+
+The Pure Storage FlashBlade receiver, receives metrics from Pure Storage internal services hosts.
```

Comment:
The Pure Storage FlashBlade receiver, receives metrics from Pure Storage FlashBlade via the [Pure Storage FlashBlade OpenMetrics Exporter](https://github.com/PureStorage-OpenConnect/pure-fb-openmetrics-exporter)